src/ha_addon_sunsynk_multi/options.py

```diff
@@ -52,24 +65,30 @@ class Options:
 
     _LISTS = {"inverters": InverterOptions, "schedules": Schedule}
 
-    mqtt_host: str = ""
-    mqtt_port: int = 0
-    mqtt_username: str = ""
-    mqtt_password: str = ""
-    number_entity_mode: str = "auto"
-    prog_time_interval: int = 15
-    inverters: list[InverterOptions] = []
-    sensor_definitions: str = "single-phase"
-    sensors: list[str] = []
-    sensors_first_inverter: list[str] = []
-    read_allow_gap: int = 10
-    read_sensors_batch_size: int = 60
-    schedules: list[Schedule] = []
-    timeout: int = 10
-    debug: int = 0
-    driver: str = "umodbus"
-    manufacturer: str = "Sunsynk"
-    debug_device: str = ""
+    mqtt_host: str = os.getenv("MQTT_HOST", "")
+    mqtt_port: int = int(os.getenv("MQTT_PORT", "0"))
+    mqtt_username: str = os.getenv("MQTT_USERNAME", "")
+    mqtt_password: str = os.getenv("MQTT_PASSWORD", "")
+    number_entity_mode: str = os.getenv("NUMBER_ENTITY_MODE", "auto")
+    prog_time_interval: int = int(os.getenv("PROG_TIME_INTERVAL", "15"))
+    inverters: list[InverterOptions] = [
+        InverterOptions(**{k.lower(): v for k, v in inv.items()})
+        for inv in loads(os.getenv("INVERTERS", "[]"))
+    ]
+    sensor_definitions: str = os.getenv("SENSOR_DEFINITIONS", "single-phase")
+    sensors: list[str] = loads(os.getenv("SENSORS", "[]"))
```

Comment:
We could potentially support simple comma separated strings here? split instead of loads

Reply:
Yes, but I think this is more consistent with other options. In my opinion it could be added as another way to input a list, but shouldn't replace the json format.